pkg/application/application.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
 )
 
+type JiraAPIResourceInterace interface {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type JiraAPIResourceInterace should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)